design(website): Change version dropdown design

fhennig opened the pull request:
resolves #3738
resolves https://github.com/loculus-project/loculus/issues/2935


preview URL: 

### Summary
<!-- Add a few sentences describing the main changes introduced in this PR -->
- Changes the width of the popup and some texts and colors of the labels.

#### Manual testing
I looked at the changes and also the changes on the big version page. You can see what I did also in the video below.

### Screenshot


https://github.com/user-attachments/assets/62224154-223c-4bee-b17d-2a12e4be48ff


### PR Checklist
<!-- Check completed items of strikethrough irrelevant items (using ~~text~~) -->
- ~~All necessary documentation has been adapted.~~
- [x] The implemented feature is covered by appropriate, automated tests.
- [x] Any manual testing that has been done is documented (i.e. what exactly was tested?)

fhennig commented:
Not sure all the different versions that were discussed but I think this is maybe a good first step? at least the width thing is definitely fixed.

theosanderson commented:
Agree that this looks a useful first step. It would be great to keep some issue open to track the remaining point which would be something like:
- change "revised" and "revoked" to "Previous version"
- When the Latest version is a revocation, probably style "Latest version" in red with "Sequence revoked" if we don't already

fhennig commented:
Happy to just do this in this PR and I think there's not rush to merge it as is, I just feel like there might be other opinions around. I think I'm going to ask on Slack what the others think.

fhennig commented:
I've made the changes! I've also updated the video.

I've added a test stub, I think I'm gonna add some quick tests as well - but other than that maybe this is fine?

theosanderson reviewed (approved):
LGTM! Thanks!
